boefjes/boefjes/plugins/kat_ssdp/schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Arguments",
+  "type": "object",
+  "properties": {
+    "SEARCHTARGET": {
+      "title": "SEARCHTARGET",
+      "type": "string",
+      "description": "What search target to use. Defaults to ssdp:all, needs to be a valid ssdp query."
+    },
+    "TIMEOUT": {
+      "title": "TIMEOUT",
+      "maxLength": 3,
+      "type": "string",
```

Comment:
Should be an integer

Reply:
Should I also remove the default value inside `boefje.py` as well?

Reply:
I think it's good to have the default values there as well because you can still have it running without configuring these arguments resulting in empty environment vars